### ISSUE TITLE
Syndieborgs have no brains

### ISSUE THREAD

**Issue by ArcLumin** [HippieStationArchive/HippieStation13-1#1752]
I found someone who was changed into a syndieborg by a chaos magicarp. When I deconstructed the syndieborg, I found the MMI, properly labeled with their original name, but with no brain inside. Sprite had no brain, could not remove the non-existent brain (Gave a no brain in the mmi message), but functioned otherwise like a normal MMI; he could talk, his consciousness was still in it, and I could put him into other borg bodies.

**Comment by TDcoolguy300** [comment]
This is due to the brain being a physical object in it, and when changed by by the staff of Change it doesnt transfer the actual brain item to the MMI

**Comment by Nebyab** [comment]
This isn't a bug, moreso just a feature of how the game works.

**Comment by ArcLumin** [comment]
No, it is a bug when you can put ANOTHER brain inside it, making it two people in one MMI

**Comment by Pyko1** [comment]
i thought this was fixed, is it?

**Comment by ArcLumin** [comment]
If I recall, nope
